src/main/scala/nl.knaw.dans.easy.solr4files/ApplicationWiring.scala

```diff
@@ -68,35 +68,33 @@ class ApplicationWiring(configuration: Configuration)
   private def updateStores(storeNames: Seq[String]): Try[Seq[FeedBackMessage]] = {
     storeNames
       .map(initSingleStore)
-      .collectResults.recoverWith {
-      case t: CompositeException => throw new Exception(s"Tried to update ${ storeNames.size } stores, ${ t.getMessage() }", t)
-    }
+      .collectResults
+      .recoverWith { case t: CompositeException =>
+        throw new Exception(s"Tried to update ${ storeNames.size } stores, ${ t.getMessage() }", t)
+      }
   }
 
   private def updateBags(storeName: String, bagIds: Seq[String]): Try[Seq[FeedBackMessage]] = {
     bagIds
       .map(uuid => update(storeName, uuid))
-      .collectResults.recoverWith {
-      case t: CompositeException => throw new Exception(s"Tried to update ${ bagIds.size } bags, ${ t.getMessage() }", t)
-    }
-  }
-
-  private def createDocs(bag: Bag, ddm: DDM, files: Seq[FileItem]): Try[FeedBackMessage] = {
-    val results = files.map(fileItem => createDoc(bag, ddm, fileItem))
-    collectMixedResults(bag.bagId, results)
+      .collectResults
+      .recoverWith { case t: CompositeException =>
+        throw new Exception(s"Tried to update ${ bagIds.size } bags, ${ t.getMessage() }", t)
+      }
   }
 
-  private def collectMixedResults(bagId: String, results: Seq[Try[FeedBackMessage]]) = {
-    val nrOfFailures = results.count(_.isFailure)
-    val successMessages = results.filter(_.isSuccess).map(_.get)
-    val count = successMessages.count(_.startsWith("update retried"))
-    val stats = s"Bag $bagId: updated ${ successMessages.size } files, $count of them without content"
-    logger.info(stats)
-    if (nrOfFailures == 0)
-      Success(stats)
-    else {
-      val t = results.collectResults.failed.get
-      Failure(new Exception(s"$stats, another ${ t.getMessage }", t))
+  private implicit class mixedResults(left: Seq[Try[FeedBackMessage]]) {
```

Comment:
put an extra `extends AnyVal` here for some more optimization: https://docs.scala-lang.org/overviews/core/value-classes.html#extension-methods

Reply:
causes compiler errors